packages/cactus-plugin-bungee/tsconfig.json

```diff
@@ -0,0 +1,41 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "composite": true,
+    "outDir": "./dist/lib/", /* Redirect output structure to the directory. */
+    "declarationDir": "dist/lib",
+    "resolveJsonModule": true,
+    "allowJs": false,
+    "skipLibCheck": true,
+    "rootDir": "./src/",
+    "tsBuildInfoFile": "../../.build-cache/cactus-plugin-bungee.tsbuildinfo",
+    "moduleResolution": "Node16"
```

Comment:
@eduv09 The current default here is `node` so if you set it to Node16 it might (as in most likely) break the build and some other things. If there is a specific reason why this has to be set to this then let's talk about it and figure out a solution together. Me and @outSH spent countless hours working through issues related to ESM resolutions for example and 99% of the time between the two of us there will be some solution that we can recommend without having to set this to Node16 and potentially making trouble for the build process.

Reply:
> @eduv09 The current default here is `node` so if you set it to Node16 it might (as in most likely) break the build and some other things. If there is a specific reason why this has to be set to this then let's talk about it and figure out a solution together. Me and @outSH spent countless hours working through issues related to ESM resolutions for example and 99% of the time between the two of us there will be some solution that we can recommend without having to set this to Node16 and potentially making trouble for the build process.

@eduv09 Please see my comments on the bottom, I've investigated this in the meantime and found a solution.